vips/vip-255/bscmainnet.ts

```diff
@@ -0,0 +1,70 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+import { NORMAL_TIMELOCK } from "../../src/vip-framework";
+
+const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+const VAI = "0x4BD17003473389A42DAF6a0a729f6Fdb328BbBd7";
+const V2_LP = "0xD94FeFc80a7d10d4708b140c7210569061a7eddb";
+const V2_LP_BALANCE = parseUnits("176508.405573652762506292", 18);
+const TIMELOCK_VAI_BALANCE = parseUnits("1598.336577715436018042", 18);
+const LIQUIDITY_MOVER = "0x6F9947f15896169C122EaF621628C437B7d3583e";
```

Comment:
The difference is that this contract moves liquidity to a dynamic range (centered at current price), while V3Migrator moves it to a fixed range specified by (tickLower, tickUpper). I tried the fixed range approach, and the liquidity we'll be adding will depend on the price a lot because all ticks above or below the current price will be denominated in a currency different from the one we receive from v2. Maybe we could still reuse V3Migrator somehow, e.g. by inheritance, but tbh PCS contracts don't really work well with the core repo, I figured it's easier to just reimplement it :D